utils/useful.py

```diff
@@ -1,38 +1,59 @@
-import copy
-import inspect
-import discord
-import datetime
-import ctypes
-import traceback
-import sys
+from __future__ import annotations
```

Comment:
hoe really put this just because of 1 typing issue

Reply:
literally dont care